src/linechart/DataSeries.jsx

```diff
@@ -12,129 +11,90 @@ module.exports = React.createClass({
 
   propTypes: {
     data: React.PropTypes.array,
-    seriesName: React.PropTypes.string,
-    interpolationType: React.PropTypes.string,
     fill: React.PropTypes.string,
     xAccessor: React.PropTypes.func,
     yAccessor: React.PropTypes.func,
-    displayDataPoints: React.PropTypes.bool
+    colorAccessor: React.PropTypes.func,
+    interpolationType: React.PropTypes.string
   },
 
   getDefaultProps() {
     return {
       data: [],
-      seriesName: "",
-      interpolationType: 'linear',
       fill: '#fff',
```

Comment:
The `fill` prop is no longer needed since we are using `colors` and `colorAccessor`